pages/proposals/language/English.json

```diff
@@ -6,7 +6,7 @@
     "background": {
       "header": "Welcome to the MozFest 2017 Call for Proposals",
       "body": [
-        "MozFest sessions cover a wide range of topics, but share two important qualities: they are <em>interactive</em> and <em>inclusive</em>.",
+        "MozFest sessions cover a wide range of topics, but share two important qualities: they are <em>interactive</em> and <em>inclusive</em>. We welcome workshops and exhibits, and often shy away from sessions that are exclusively lectures and slide presentations.",
```

Comment:
Given its connotations, "shy away" is potentially problematic phrasing for us to use. Would the following be acceptable phrasing?

"We welcome workshops and exhibits, and strongly favor these over sessions that are exclusively lectures and slide presentations.",